simpletransformers/question_answering/question_answering_utils.py

```diff
@@ -1069,7 +1069,7 @@ def get_best_predictions(
                 all_predictions[example.qas_id] = best_non_null_entry.text
         all_nbest_json[example.qas_id] = nbest_json
 
-    all_best = [{"id": id, "answer": answers[0]["text"]} for id, answers in all_nbest_json.items()]
+    all_best = [{"id": id, "answer": answers[0]["text"], "prob": round(answers[0]["probability"],4)} for id, answers in all_nbest_json.items()]
     return all_best
 
```

Comment:
Shall we return the probabilities as a separate value rather than including it in the list of answers itself? Then we can return the answers list and the probabilities list from the `predict()` function (so that `predict()` has two return values). This is to keep the `QuestionAnsweringModel` consistent with the other models.